lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.html

```diff
@@ -1,49 +0,0 @@
-<form [formGroup]="form" novalidate (ngSubmit)="apply(form.value, form.valid)">
-    <mat-form-field>
-        <mat-label>{{ 'SEARCH.FILTER.RANGE.SELECT-FROM-DATE' | translate }}</mat-label>
-        <input matInput
-                (input)="forcePlaceholder($event)"
-                [formControl]="from"
-                [errorStateMatcher]="matcher"
-                placeholder="{{ 'SEARCH.FILTER.RANGE.FROM-DATE' | translate }}"
-                [attr.aria-label]="'SEARCH.FILTER.RANGE.FROM-DATE' | translate"
-                [matDatepicker]="fromDatepicker"
-                [max]="fromMaxDate"
-                (dateChange)="onChangedHandler($event, from)"
-                data-automation-id="date-range-from-input">
-        <mat-datepicker-toggle matSuffix [for]="fromDatepicker" data-automation-id="date-range-from-date-toggle"></mat-datepicker-toggle>
-        <mat-datepicker #fromDatepicker color="accent"></mat-datepicker>
-        <mat-error *ngIf="from.invalid" data-automation-id="date-range-from-error">
```

Comment:
this variable seems still referencing to that removed element, please remove that variable as this element does not exist anymore: 
<img width="1008" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/4a0506b0-f96f-44e5-a9c5-7f395159c32f">

Reply:
Removed date-range-filter.page.ts